CHANGELOG.md

```diff
@@ -29,10 +29,22 @@ All notable changes to this project will be documented in this file.
 - *(collection)* Remove completed todos from sensitization help page
 - *(collection)* Add first pass with food items
 - *(collection)* Update sensitization first pass button todo comment
+- *(changelog)* Update readme
+- *(changelog)* Update readme
+- *(changelog)* Update with further readme changes
+- *(changelog)* Update with fvm build and readme changes
 - *(license)* Add copyright year and owner
+- *(readme)* Add unix-like systems note again
+- *(readme)* Update contributors and acknowledgments
+- *(readme)* Add note about assuming a unix-like system
+- *(readme)* Add minor updates
+- *(readme)* Add development instructions and update authors
+- *(readme)* Add note about fvm doctor and delete unnecessary command
+- *(readme)* Update fvm and developement instructions
 - *(readme)* Update to fvm version 2.5.1
 - *(readme)* Update style in title and oneliner
-- *(readme)* Update readme with new name, correct style and typos- *(No Category)* Use git-cliff for changelog generation and update changelog
+- *(readme)* Update readme with new name, correct style and typos- *(No Category)* Move development instructions from readme to a standalone file in docs
```

Comment:
Actually messed up here as this didn't resolve the conflict but the subsequent commit has 💁🏾‍♀️.